easybuild/easyconfigs/i/icc/icc-2015.3.187-GCCcore-4.9.3-binutils-2.25.eb

```diff
@@ -0,0 +1,27 @@
+name = 'icc'
+version = '2015.3.187'
+
+homepage = 'http://software.intel.com/en-us/intel-compilers/'
+description = "C and C++ compiler from Intel"
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['l_ccompxe_%(version)s.tgz']
+
+gcc = 'GCCcore'
+binutilsver = '2.25'
+gccver = '4.9.3-binutils-%s' % binutilsver
```

Comment:
good point

there's a problem with this easyconfig though, it doesn't provide what it claims to (based on the filename):

```
ERROR: easyconfig file icc-2015.3.187-GCCcore-4.9.3-binutils-2.25.eb does not contain module icc/2015.3.187-GCCcore-4.9.3-binutils-2.25 (mods: ['icc/2015.3.187'])
```

missing `versionsuffix`?

@ocaisa: I'll play around with this a bit, and send you a PR soonish.